lib/logger.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+class Logger {
+    constructor() {}
+
```

Comment:
All these methods output error to the console? Since it's server side, I guess I'm curious as to why as opposed to a file?

Reply:
The default logger is really simple and just outputs to the console, but I added an accessor so that an external logger can be passed in. We will supply paper-handlebars with a more robust logger that does what we want for our infrastructure.